src/integration-test/groovy/au/org/ala/fieldcapture/RlpReportingSpec.groovy

```diff
@@ -417,4 +418,213 @@ class RlpReportingSpec extends StubbedCasSpec {
+                'koRLP_-_Controlling_access',
+                'koRLP_-_Pest_animal_management',
+                'koRLP_-_Management_plan_development',
+                'koRLP_-_Debris_removal',
+                'koRLP_-_Erosion_Management',
+                'koRLP_-_Maintaining_feral_free_enclosures',
+                'koRLP_-_Establishing_ex-situ_breeding_programs',
+                'koRLP_-_Establishing_Agreements',
+                'koRLP_-_Establishing_monitoring_regimes',
+                'koRLP_-_Farm_Management_Survey',
+                'koRLP_-_Fauna_survey',
+                'koRLP_-_Fire_management',
+                'koRLP_-_Flora_survey',
+                'koRLP_-_Habitat_augmentation',
+                'koRLP_-_Identifying_sites',
+                'koRLP_-_Improving_hydrological_regimes',
+                'koRLP_-_Improving_land_management_practices',
+                'koRLP_-_Disease_management',
+                'koRLP_-_Negotiations',
+                'koRLP_-_Obtaining_approvals',
+                'koRLP_-_Pest_animal_survey',
+                'koRLP_-_Plant_survival_survey',
+                'koRLP_-_Project_planning',
+                'koRLP_-_Remediating_riparian_and_aquatic_areas',
+                'koRLP_-_Weed_treatment',
+                'koRLP_-_Revegetating_habitat',
+                'koRLP_-_Site_preparation',
+                'koRLP_-_Skills_and_knowledge_survey',
+                'koRLP_-_Soil_testing',
+                'koRLP_-_Emergency_Interventions',
+                'koRLP_-_Water_quality_survey',
+                'koRLP_-_Weed_distribution_survey']
+
+        when: "We complete the form and save, marking optional sections as not applicable"
+        hideFloatingToolbar()
+        field('whsRequirementsMet').value('Met requirements')
+        field('variationSubmitted').value('No')
+        field('meriOrWorkOrderChangesRequired').value('No')
+        getFormSections().each {
+            if (isOptional(it)) {
+                markAsNotApplicable(it)
+            }
+        }
+        restoreFloatingToolbar()
+        markAsComplete()
+        save()
+        exitReport()
+
+        then: "then the project start date cannot be change as one of the reports has data"
+        waitFor { at RlpProjectPage }
+
+        and: ""
+        assert projectReports.projectStartDate.empty == true
+        assert projectReports.generateButton.empty == true
+    }
+
+    def "A read only user will not be able to generate reports and change the project start date " () {
+
+        setup:
+        String projectId = '1'
+        loginAsReadOnlyUser(browser)
+
+        when: "login as read only"
+        to ProjectIndex, projectId
+
+        then:
+        waitFor { at ProjectIndex }
+
+        when:
+        reportingTab.click()
+
+        then:
+        waitFor 20, {
+            projectReports.displayed
+        }
+
+        and:
+        assert projectReports.projectStartDate.empty == true
```

Comment:
We don't need the assert here - anything in then:/and: is automatically an assertion.